docs/aca/10-aca-iac-bicep/index.md

```diff
@@ -2,7 +2,208 @@
+
+### ARM Templates in Azure
+
+ARM templates are files that define the infrastructure and configuration for your deployment, the template uses declarative syntax, which lets you state what you intend to deploy without having to write the sequence of programming commands to create it.
+
+Within Azure there are 2 ways to create IaC, we can use the [JSON ARM templates](https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/overview) or [Bicep](https://docs.microsoft.com/en-us/azure/azure-resource-manager/bicep/overview?tabs=bicep) (domain-specific language). From past experience, using JSON ARM templates in real-world scenarios tends to be complex to manage and maintain especially when the project grows and the number of components and dependencies increases.
+
+Using Bicep is simpler and easier to work with, you can be more productive compared to ARM templates, it is worth mentioning that Bicep code will be compiled into ARM templates eventually, this process called "Transpilation".
+
+To learn more about Bicep, I highly recommend checking Microsoft learn website [Fundamentals of Bicep.](https://docs.microsoft.com/en-us/training/paths/fundamentals-bicep/)
+
+![aca-arm-bicep](../../assets/images/10-aca-iac-bicep/aca-bicep-l.jpg)
+
+### Build the Infrastructure as Code using Bicep
+
+Let's get started by defining the Bicep modules needed to create the Infrastructure code, what we want to achieve by the end of this module is to have a new resource group containing all the resources and configuration (connection strings, secrets, env variables, dapr components, etc..) we used to build our solution, we should have a new resource group which contains the below resources.
+![aca-resources](../../assets/images/10-aca-iac-bicep/aca-rescources.jpg)
+
+!!! note
+    To simplify the execution of the module, we will assume that the azure resource "Azure Container Registry" is already provisioned and it contains the latest images of the 3 services. We will not provision Azure Container Registry part of this Bicep modules.
+
+#### 1. Add the needed extension to VS Code
+You need to install an extension named [Bicep](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-bicep), This extension will simplify building Bicep files as they will offer IntelliSense, Validation, listing all available resource types, etc..
+
+#### 2. Define an Azure Container Apps Environment
+Add a new folder named `Bicep` on the root project directory, then add another folder named `Modules`. Add a new file named `container-apps-environment.bicep`. The content of the file can be found on this [link.](https://github.com/Azure/aca-dotnet-workshop/blob/main/bicep/modules/container-apps-environment.bicep).
+
+What we've added to this file is the following:
```

Comment:
Bullet list is bit off in the preview url https://azure.github.io/aca-dotnet-workshop/pr-preview/pr-54/aca/10-aca-iac-bicep/#2-define-an-azure-container-apps-environment, maybe it need an extra <kbd>Enter</kbd>

Reply:
Done.